project/npda/templates/npda/visit_form.html

```diff
@@ -29,7 +29,12 @@
       {% endfor %}
 
       <div role="tablist" class="tabs tabs-bordered">
-        <input type="radio" name="my_tabs_1" role="tab" class="tab" aria-label="Routine&nbsp;Measurements" checked/>
+        <input type="radio" name="my_tabs_1" role="tab" class="tab font-bold
+          {% for category in routine_measurements_categories %} 
```

Comment:
We could also consider passing an extra variable through context to avoid the iteration here? something like `routine_measurements_category_has_errors`? Same for annual review below.

If we stick with calculating it in the template please do tab the logic appropriately and use new lines as you would for anything else

Reply:
Actioned! Thanks for the suggestion, is much tidier now. Before we merge I'd like to make some changes to the Routine Measurements form so that the errors there can be localised by the user a bit more easily.